subs/pantry/src/Pantry/Archive.hs

```diff
@@ -375,7 +381,7 @@ parseArchive pli archive fp = do
         Left e -> throwIO $ UnsupportedTarball loc $ T.pack e
         Right safeFiles -> do
           let toSave = Set.fromList $ map (seSource . snd) safeFiles
-          blobs <-
+          (blobs :: Map FilePath BlobKey)  <- --come here
```

Comment:
What does this comment mean? Is type here required?

Reply:
Fixed.